tests/misc/CachingTest.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+import re
+import os
+import time
+
+from pyprint.NullPrinter import NullPrinter
+
+from coalib.misc.Caching import FileCache
+from coalib.misc.CachingUtilities import (
+    get_settings_hash, settings_changed, update_settings_db,
+    pickle_load, pickle_dump
+    )
```

Comment:
Argh, missed my eye (I must have deleted a few imports in that line and forgotten about the parens)